### ISSUE TITLE
verdi_script_two_vars script

### ISSUE THREAD

**Issue by lizadams** [CEMPD/VERDI#58]
1. \- [ ] Getting an error in eclipse console when running script to load two variables and create tile plots for each one:
   into runApplicationRunnable
   Exception in thread "Thread-4" java.lang.NullPointerException
   at saf.core.ui.dock.DefaultDockableFrame.isMinimized(DefaultDockableFrame.java:108)
   at anl.verdi.core.VerdiGUI.isHidden(VerdiGUI.java:240)
   at anl.verdi.plot.gui.FastTilePlot$5.run(FastTilePlot.java:395)
   at java.lang.Thread.run(Thread.java:745)
   did appRunner.run, returning
   back from runApplicationRunnable
2. \- [ ] Getting an error when have two fast tile plots and re-arranging them, moving the second tab to be first when I am turning the county map layer on and off.
   Running script within eclipse that loads two variables.
   Exception in thread "Thread-5" java.util.ConcurrentModificationException
   at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:901)
   at java.util.ArrayList$Itr.next(ArrayList.java:851)
   at gov.epa.emvl.Mapper.draw(Mapper.java:131)
   at anl.verdi.plot.gui.FastTilePlot$5.run(FastTilePlot.java:614)
   at java.lang.Thread.run(Thread.java:745)
   _NSJVMLoadLibrary: NSAddLibrary failed for /libjawt.dylib
   JavaVM FATAL: lookup of function JAWT_GetAWT failed. Exit
3. \- [ ] To reproduce these errors outside of Eclipse, the equivalent command line options are:
   ./verdi.command -f "../../data/model/CCTM46_P16.baseO2a.36k.O3MAX" -s "NO[1]+NO2[1]" -ts 12 -levelrange 2 1 -mapName ../../data/map_county/tl_2015_us_county.shp -configFile "../../data/configs/o3_10bin.cfg" -g tile -s "O3[1]" -g tile

**Comment by lizadams** [comment]
using VERDI_2.0_beta_linux64_20190623.tar.gz
the following script causes an error
```
./verdi.sh -f "$cwd/data/model/CCTM46_P16.baseO2a.36k.O3MAX" -s "NO[1]+NO2[1]" -ts 12 -g tile -s "O3[1]" -g tile
```
It works if you only try to create the first tile plot, but not if you try to create two.

./verdi.sh -f "$cwd/data/model/CCTM46_P16.baseO2a.36k.O3MAX" -s "NO[1]+NO2[1]" -ts 12 -g tile -s "O3[1]" -g tile
2019.06.25 13:01:43.490 [Thread-5] ERROR anl.verdi.plot.gui.FastTilePlot - Error rendering FastTilePlot
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:635) ~[?:1.7.0_79]
	at java.util.ArrayList.get(ArrayList.java:411) ~[?:1.7.0_79]
	at bibliothek.gui.dock.station.stack.DefaultStackDockComponent.getDockableAt(Unknown Source) ~[dockingFramesCore.jar!/:?]
	at saf.core.ui.dock.DefaultDockableFrame.isHidden(DefaultDockableFrame.java:135) ~[saf.core.ui.jar!/:?]
	at anl.verdi.core.VerdiGUI.isHidden(VerdiGUI.java:255) ~[core.jar!/:?]
	at anl.verdi.plot.gui.FastTilePlot$5.run(FastTilePlot.java:415) [core.jar!/:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]

**Comment by lizadams** [comment]
confirmed this issue is fixed using VERDI 2.1.5 2024-01-03 on mac
